src/CMakeLists.txt

```diff
@@ -231,6 +233,7 @@ endif()
 if( MIOPEN_BACKEND MATCHES "OpenCL" OR MIOPEN_BACKEND STREQUAL "HIPOC" OR MIOPEN_BACKEND STREQUAL "HIP")
     file(GLOB_RECURSE COMPOSABLE_KERNEL_INCLUDE "kernels/composable_kernel/include/*/*.hpp")
     file(GLOB_RECURSE COMPOSABLE_KERNEL_SOURCE "kernels/composable_kernel/src/*/*.cpp")
+    file(GLOB_RECURSE COMPOSABLE_KERNEL_DYNAMIC_SOURCE "kernels/composable_kernel/asm/*.s")
```

Comment:
Don't you want to put in a separate folder, considering there is not dependency on composable kernel? It's up to you

Reply:
I think it's reasonable. then let me re-arrange into a separate folder name `kernels/dynamic_igemm/`